### ISSUE TITLE
PHP warning in WP Debug Log

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#38]
“[20-Jul-2021 14:58:37 UTC] PHP Warning: array_key_exists() expects parameter 2 to be array, null given in /var/www/abundance.ca/htdocs/wp-content/plugins/accessibility-checker/includes/rules/color_contrast_failure.php on line 715”
“[20-Jul-2021 14:58:41 UTC] PHP Notice: Undefined index: :root in /var/www/abundance.ca/htdocs/wp-content/plugins/accessibility-checker/includes/rules/color_contrast_failure.php on line 715”

Additional information on [support ticket](https://products.equalizedigital.com/wp-admin/post.php?post=1507&action=edit).

**Comment by SteveJonesDev** [comment]
Fixed since version 1.3.2

c0133d3d07e5aa2552da54246af21b139b841a56
